### PR TITLE
[Publications] updated module help text

### PR DESCRIPTION
**Pull request by knoll-alex**
New copy for Publications module help text

#### Testing instructions

1. Review copy
2. See screenshot for rendering

#### Links to related tickets

* https://github.com/aces/Loris/issues/5576

Screenshot 2019-11-14 14 14 28

### CODE REVIEW

**Comment by knoll-alex** [comment]
Note - in the menu at the top of LORIS, this module is labelled Publications, but labelled as Publication (no s) everywhere else

**Comment by christinerogers** [comment]
@Jkat  -- could you take 30 secs to review this update to the Publication module Help text (as the module expert) ? 
Please [review this markdown](https://github.com/aces/Loris/blob/001c58a8994aed212895014f43f75870eef63314/modules/publication/help/publication.md)  (not the out of date screenshot above)

File: modules/publication/help/publication.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
+In the **Browse** tab, use the selection filters to search for a project of interest. 
 
+You can submit a new proposal in the **Propose a Project** tab, and upload specific files to associate with it. 
 
-Navigate to the "__Propose a Project__" tab for proposing research projects based on the data required. You may upload publication media which may then be reviewed by study PIs.
+Pay special attention to user permissions when submitting proposals. 
+Note that PIs can designate other users to edit and later update the project or publication information. Only users affiliated with the proposal can delete publications. 
 
-You should specify which specific variables of interest are relevant to your research so that the reviewers can determine if they should grant access to the data.
+You should also make sure to specify which variables of interest are relevant to your research, so reviewers can properly determine if they should grant access to the data.
```
Comment (PapillonMcGill) [inline modules/publication/help/publication.md]:
```suggestion
You should also ensure to specify which variables of interest are relevant to your research, so reviewers can properly determine if they should grant access to the data.
```

Comment (christinerogers) [reply modules/publication/help/publication.md]: Thanks @PapillonMcGill - we're shifting the style away from `ensure` to make it more friendly / easily readable.

Comment (PapillonMcGill) [reply modules/publication/help/publication.md]: @christinerogers In my opinion, it look more professional and less sloppy
I don't think going back to make sure in a good move

Comment (knoll-alex) [reply modules/publication/help/publication.md]: What about removing that altogether? Instead: "It's important to specify which variables of interest are relevant to your research..."

Comment (christinerogers) [reply modules/publication/help/publication.md]: I think "Important to specify" is better than "make sure" -- though I know "make sure" has already made it into a lot of the Help texts already merged.

**Comment by christinerogers** [comment]
> The writing looks good to me. However, I think @LeighMac would be in a better position to validate this text since this module is being actively used by IBIS (and not by CCNA unfortunately even though it was originally requested by CCNA)

Thanks @mohadesz  - please leave an Approval review if you're comfortable with that.  thanks for also tagging Leigh.

**Comment by christinerogers** [comment]
2 approvals - can be merged @driusan

**Pull request merged**